jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/Constants.java

```diff
@@ -43,4 +46,8 @@ private Constants() {}
   public static final AttributeKey<String> ATTR_ARENA_NAME = AttributeKey.stringKey("arena");
   public static final AttributeKey<String> ATTR_NETWORK_MODE = AttributeKey.stringKey("mode");
   public static final AttributeKey<String> ATTR_USAGE = AttributeKey.stringKey("usage.type");
+  public static final AttributeKey<String> ATTR_DAEMON = AttributeKey.stringKey(DAEMON);
```

Comment:
This should be a boolean attribute (see the [JMX based implementation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/Threads.java#L36) for comparison)

Reply:
changed!